docs/dxp/7.x/en/site-building/site-appearance/developer-guide/themes/introduction-to-themes.md

```diff
@@ -0,0 +1,50 @@
+# Introduction to Themes
+
+## What is a Theme?
+
+* What is a theme? Is it needed? What can you do with it?
+* Maybe link to the Getting Started article for changing a Site's theme to ground discussion
+
+Maybe reference the "theme anatomy", and if possible show a picture (otherwise link to an article in the same vein)
+
+## Possible Section on Just Using Themes in general in Liferay UI?
+
+[Screenshot of page with one of the non-classic themes?]
+
+Not sure if there's enough content past, how to change a theme (with link), for a whole section... but if so, put that here
```

Comment:
I see this section closely connected to the "setting a theme for my page/site". In that case, this is covered in [Configuring Page Sets](https://help.liferay.com/hc/es/articles/360029132671-Configuring-Page-Sets) article.